### ISSUE TITLE
nginx: [emerg] host not found in upstream "<PHP_SOCKET>"

### ISSUE THREAD

**Issue by aahmadts** [webdevops/Dockerfile#187]
webdevops/php-nginx:5.6
for some reason nginx doesn't run anymore. I'm not sure if this is because of some updates or because it is Monday :p .Everything was working great but now the container doesn't run and I get this.

`
nginx -q
nginx: [emerg] host not found in upstream "<PHP_SOCKET>" in /opt/docker/etc/ngin
x/conf.d/10-php.conf:2
`

where `/opt/docker/etc/nginx/conf.d/10-php.conf` looks like:

`upstream php {
    server <PHP_SOCKET>;
}`

**Comment by aahmadts** [comment]
works now, but I have no idea what the problem was.

**Comment by pthomas931** [comment]
I am having this issue today. I am using the default php and nginx configs.

**Comment by revolter** [comment]
Same issue, but I didn't fix it.

**Comment by sarenodev** [comment]
TL;DR: Use `/entrypoint supervisord` instead of just `supervisord`. Turns out `/entrypoint` does some magic behind the scenes to replace placeholders with actual values.
Related: https://github.com/webdevops/Dockerfile/issues/187#issuecomment-476839679.
